docs/content/docs/build/containers.md

```diff
@@ -107,72 +114,48 @@ const { container, services } =
     await client.getContainer(/*container id*/, schema);
 ```
 
-### Deleting a container
-
-Deleting a container is a service-specific feature, so you should refer to the documentation associated with the specific Fluid service you are using. See [Available Fluid Services]({{< relref "service-options.md" >}}) for more information about Fluid service options.
-
-## Container lifecycle and manipulation
-
-### attached/detached
-
-Newly created containers begin in a *detached* state.
-A detached container is not connected to the Fluid service and no data is shared with other clients.
-This is an ideal time to create initial data in your Fluid data model.
-
-An *attached* container, conversely, **is** connected to the Fluid service, and data changes are shared with the other clients.
-An existing container being loaded by a client will always initially be in an attached state.
+## Container lifecycle & events
 
-### connected/disconnected
+### Connection state
 
 The container exposes the connected state of the client and emits connected and disconnected events to notify the caller if the underlying connection is disrupted.
 Fluid will by default attempt to reconnect in case of lost/intermittent connectivity.
 However, `connect()` and `disconnect()` can be used to manually control the connection policy.
 
-Note about this code:
-
-- `user.on("idle")` and `user.on("active")` are hypothetical event callbacks which fire if a user becomes idle or active respectively. They are for demonstration purposes only and are not provided by FluidFramework.
-
 ```typescript {linenos=inline}
 const connectionState = container.connectionState;
 
 container.on("disconnected", () => {
-    // handle disconnected
-    // prevent the user from editing to avoid data loss
+    // prevent user edits to disable data loss.
 });
 
 container.on("connected", () => {
-    // handle connected
-    // enable editing if disabled
+    // enable editing if disabled.
 });
 
+// Hypothetical event firing when user becomes idle. It is for demonstration purposes only.
 user.on("idle", () => {
     // Disconnect the container when idle to prevent unnecessary network traffic/COGS
     container.disconnect();
 });
 
+// Hypothetical event firing when user becomes active. It is for demonstration purposes only.
 user.on("active", () => {
     // Connect the container when active to resume op processing
     container.connect();
 });
 ```
 
-### dispose
+Notes:
+- Connection is established by default whenever container is loaded or attached.
```

Comment:
```suggestion
- Connection is established by default whenever an existing container is loaded or a new container is attached.
```

"Loaded" isn't a state, right? This is just referring to existing vs. new containers?